mainview.py

```diff
@@ -1,5 +1,5 @@
-from workspacemanager.WorkspaceView import WorkspaceView
-from MainPresenter import MainPresenter
+from main_presenter import MainPresenter
```

Comment:
Looks like MainPresenter lives outside the reset of the presenters?

Also. I'm still seeing a lot of inconsistencies with the casing. You use lower case for modules in some places but not in others.